README.md

```diff
@@ -1,10 +1,13 @@
-# Parallel Programming on Discovery
-This repository contains material for HPC Parallel Programming on Discovery
+# HPC & Parallel Computing
+This repository contains material for HPC Parallel Computing on Discovery
 training modules.
```

Comment:
@marianalevineu , maybe we should not refer to "Discovery" here and mention something like:
This repository contains material for parallel computing on NU's high performance computing cluster

Reply:
thanks @manasvita  I removed the Discovery mentions as recommended.